dali/operators/python_function/dltensor_function.h

```diff
@@ -245,30 +245,6 @@ class DLTensorPythonFunctionImpl : public Operator<Backend> {
   bool synchronize_stream_;
   bool batch_processing;
   std::vector<TensorLayout> output_layouts_;
-
- private:
-  int GetCurrBatchSize(Workspace &ws) {
```

Comment:
Getting the batch size can be done with the `InferBatchSizeFromInput`. The uniformity check is done at the operator class level, with the `EnforceUniformInput/OutputBatchSize`. And as for the check of requested batch size, it is set uniformly `batch_sizes_.resize(NumOutput(), batch_size);`.